### DISCUSSION TITLE
misconfigured keyboard, when pressing the "/" key

### DISCUSSION THREAD

**Discussion by wendelsilva** [neutrinolabs/xrdp#3224] [Q&A]
misconfigured keyboard, when pressing the "/" key the volume increases instead of writing the key

**Comment by matt335672** [comment]
ABNT2 keyboards are poorly supported in xrdp I'm afraid.

This is fixed in `devel`, but we don't have a release planned for it yet.

**Reply by wendelsilva** [reply]
how can i fix this error?

**Reply by matt335672** [reply]
I can't say for sure, as I don't know enough.

There's very little information you've given us. I had to look at your profile to work out you were using an ABNT2 keyboard. We can look further, but you'll need to fill in a complete issue request so we know what version of xrdp you are using, and what backend you are using.

The only sure-fire way to fix this is to build the latest devel version. That contains explicit support for ABNT2 keys.
